### DISCUSSION TITLE
버전 관리 방법 스터디

### DISCUSSION THREAD

**Discussion by mdgarden** [EasyAndBeauty/DailyReceipt-FE-WEB#147] [General]
버전 관리 하고 싶다는 꿈은 있는데 아는게 없읍니다
갠적으로는 로그인 + 투두API까지 되어있는 상태를 1.0으로 하고 싶어요

- 무엇을 기준으로 버전을 나누면 좋을지
- 버전 관리하는 구체적인 방법
- 백엔드와 프론트를 같이 관리할지 아니면 따로 가는건지

**Comment by IGhost-P** [comment]
저도 이부분에 대해서 궁금하네요.. 항상 버전 관리를 3단계로 하긴했습니다.
0.0.0 이라고 하면다면
첫번째 0 => 배포
두번째 0 => 업데이트
세번째 0 => 핫픽스

로 정하고 시작은 했었는데 실무에선 어떻게 하는지 궁금하네용
